src/app/services/technical-record/technical-record.service.ts

```diff
@@ -294,6 +294,11 @@ export class TechnicalRecordService {
       });
   }
 
+  generatePlate(techRecord?: TechRecordModel) {
+    console.log('Ping.');
```

Comment:
Can you just place a TODO comment here saying we need to come back and add an actual API call